StackExchange.Redis/StackExchange.Redis.csproj

```diff
@@ -83,6 +84,7 @@
     <Compile Include="StackExchange\Redis\HashEntry.cs" />
     <Compile Include="StackExchange\Redis\IConnectionMultiplexer.cs" />
     <Compile Include="StackExchange\Redis\InternalErrorEventArgs.cs" />
+    <Compile Include="StackExchange\Redis\InternalRegexCompiledOption.cs" />

```

Comment:
Credits to @sharwell https://github.com/dotnet/corefx/issues/340#issuecomment-120749951